src/shared/sam/cli/samCliInvokerUtils.ts

```diff
@@ -55,6 +57,25 @@ export function logAndThrowIfUnexpectedExitCode(processResult: ChildProcessResul
             message = processResult.error.message
         }
     }
+    const shortenedErrorsText = filterSamErrorMessage(processResult.stderr).join('&&')
```

Comment:
Rather than "filter", it's more like "collect all lines that look like errors". 

Did you decide against looking for `Error:`?